build-system/encrypt-secret.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
I added a lot of intentional whitespace around the secrets, and used this prompt approach (as opposed to accepting a flag/command line argument) to reduce the risk of errant characters ending up in the encrypted secret (such as newlines or quotes). It also displays the active project from gcloud, since encrypting under the wrong project would render the Cloud Build system unable to decrypt the secret (and break the deployment)